src/components/filterdialog/filterdialog.js

```diff
@@ -103,7 +104,8 @@ function updateFilterControls(context, options) {
 /**
      * @param instance {FilterDialog} An instance of FilterDialog
      */
-function triggerChange(instance) {
+function triggerChange (instance) {
```

Comment:
```suggestion
function triggerChange(instance) {
```

Reply:
Damn linter. I'm used to projects that always have a space after the function name so my linter has always been configured to do that automatically. Fixed it now.